pkgs/applications/video/pyca/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub, pycurl, dateutil, configobj, sqlalchemy, sdnotify, flask }:
+
+buildPythonApplication rec {
+  pname = "pyCA";
```

Comment:
Shouldn't this be `pyca`?

These are usually all lowercase.

Reply:
This is a little ambiguous apparently. The repository is called "pyCA", the tool itself says it's called "PyCA".

Reply:
I mean't that derivation names should be lowercase as per the documentation.

Reply:
Ahh, didn't know that. Fixed.